UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs

```diff
@@ -364,7 +364,7 @@ private static (string, ChatModifier) ProcessMessage(PlayerInfo sentByPlayer, st
 		return AddMsgColor(channels,
 			$"{chan}<b>{speaker}</b> {verb}" // [cmd]  Username says,
 			+ "  " // Two hair spaces. This triggers Text-to-Speech.
-			+ $"<size={textSize}>" + "\"" +  message + "\"" + "</size>"); // "This text will be spoken by TTS!"
+			+ $"<size=+{textSize}>" + "\"" +  message + "\"" + "</size>"); // "This text will be spoken by TTS!"
```

Comment:
what does the + sign do here?

Reply:
the + sign in TMPro adds into the size rather than statically defines a hard size

Reply:
does it look exactly the same with the default font size setting?

Reply:
yes, it doesn't follow the font scale so you need to add.